.github/workflows/integration_tests.yml

```diff
@@ -879,3 +879,166 @@ jobs:
             gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
             gh-actions-scripts/upload_helm_chart_to_google_cloud.sh dist/keptn-installer/
           fi
+
+  #######################################################################
+  # This job deploys Keptn with Keptn                                   #
+  #######################################################################
+  deploy_keptn_with_keptn:
+    name: Deploy Keptn with Keptn
+    needs: integration-test
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-20.04
+    env:
+      BRANCH: ${{ needs.integration-test.outputs.BRANCH }}
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # need to checkout "all commits" for certain features to work (e.g., get all changed files)
+          submodules: 'true'
+
+      - name: Load CI Environment from .ci_env
+        id: load_ci_env
+        uses: c-py/action-dotenv-to-setenv@v3
+        with:
+          env-file: .ci_env
+      - name: Download all artifacts from last successful build of specified branch
+        uses: dawidd6/action-download-artifact@v2.14.0
+        with:
+          # Download last successful artifact from a CI build
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: CI.yml
+          #event: push
+          workflow_conclusion: success
+          # use the following branch
+          branch: ${{ env.BRANCH}}
+          # directory where to extract artifacts to
+          path: ./dist
+
+      - name: Install Keptn CLI
+        id: install_keptn_cli
+        run: |
+          mkdir ${HOME}/downloads
+          wget "https://github.com/keptn/keptn/releases/download/${{ env.META_KEPTN_VERSION }}/keptn-${{ env.META_KEPTN_VERSION }}-linux-amd64.tar.gz" -O "${HOME}/downloads/keptn-cli.tar.gz"
+          tar -zxvf ${HOME}/downloads/keptn-cli.tar.gz
+          sudo mv keptn-*-linux-amd64 /usr/local/bin/keptn
+
+      - name: Deploy Keptn with Keptn
+        id: deploy_keptn_with_keptn
+        run: |
+          KEPTN_HELM_CHART_FILENAME=$(ls dist/keptn-installer/keptn*.tgz | head -1)
+          HELM_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/helm-service*.tgz | head -1)
+          JMETER_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/jmeter-service*.tgz | head -1)
+
+          keptn auth --endpoint=${{ secrets.META_KEPTN_API_URL }} --api-token=${{ secrets.META_KEPTN_API_TOKEN }}
```

Comment:
Separation of concerns: Please extract the `keptn auth` command into a separate step.